tests/examples/minimal_nested.txt

```diff
@@ -0,0 +1 @@
+0eNq9kEFvgzAMhf+Lz0kFVYEtf2WqUNJayBo4UWKqIsR/L4FO3WE77LLjs9/79OwZXD9iiMTSOu8/wcyvSQLzMf/BkHeJbdDidRfpmvUdTKVgAlMuCqxLvh8FdXYF4g6MxBEV0MXzDkvUse1zUqaAYIAEB1DAdshKouUUfBTtsBdYmcRXvGf8WQGykBDupE1MLY+Dw7gafmMoCD6tMc/PvsfD3rg4VEumbg3MtzMV3DCmLXB8K0/N6b2pm7Koq/rVp/gpqbcPrn+4CN2w/bL+I295AHTzpRQ=
```

Comment:
This blueprint string decodes successfully on `master` branch. What is it testing?

```sh
$ git status && cargo run --features codec-cli --release decode -d 0eNq9kEFvgzAMhf+Lz0kFVYEtf2WqUNJayBo4UWKqIsR/L4FO3WE77LLjs9/79OwZXD9iiMTSOu8/wcyvSQLzMf/BkHeJbdDidRfpmvUdTKVgAlMuCqxLvh8FdXYF4g6MxBEV0MXzDkvUse1zUqaAYIAEB1DAdshKouUUfBTtsBdYmcRXvGf8WQGykBDupE1MLY+Dw7gafmMoCD6tMc/PvsfD3rg4VEumbg3MtzMV3DCmLXB8K0/N6b2pm7Koq/rVp/gpqbcPrn+4CN2w/bL+I295AHTzpRQ=
Tue 01 Sep 2020 12:08:06 PM CEST
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
    Finished release [optimized] target(s) in 0.01s
     Running `target/release/factorio-blueprint decode -d 0eNq9kEFvgzAMhf+Lz0kFVYEtf2WqUNJayBo4UWKqIsR/L4FO3WE77LLjs9/79OwZXD9iiMTSOu8/wcyvSQLzMf/BkHeJbdDidRfpmvUdTKVgAlMuCqxLvh8FdXYF4g6MxBEV0MXzDkvUse1zUqaAYIAEB1DAdshKouUUfBTtsBdYmcRXvGf8WQGykBDupE1MLY+Dw7gafmMoCD6tMc/PvsfD3rg4VEumbg3MtzMV3DCmLXB8K0/N6b2pm7Koq/rVp/gpqbcPrn+4CN2w/bL+I295AHTzpRQ=`
{"blueprint_book":{"blueprints":[{"blueprint_book":{"blueprints":[{"blueprint":{"snap-to-grid":{"x":5,"y":1},"absolute-snapping":true,"icons":[{"signal":{"type":"item","name":"transport-belt"},"index":1}],"entities":[{"entity_number":1,"name":"transport-belt","position":{"x":2.5,"y":0.5}}],"item":"blueprint","version":281474976710656},"index":0}],"item":"blueprint-book","active_index":0,"version":281474976710656},"index":0}],"item":"blueprint-book","active_index":0,"version":281474976710656}}
```